Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Usamos la imagen base elegida
+FROM alpine:latest
+
+# Lo primero es crear un usuario sin privilegios de root y asignarle un directorio de trabajo
```

Comment:
No uséis comentarios en línea que se quedan rápidamente obsoletos. Aquí haces mucho más que eso.